src/common/marlin_server.cpp

```diff
@@ -962,6 +964,29 @@ static uint64_t _server_update_vars(uint64_t update) {
     return changes;
 }
 
+// set name of the printing gcode (for WUI)
+void marlin_server_set_gcode_name(const char *request) {
+    if (request == NULL)
+        return;
+    const char *ptr;
+    int ret = sscanf(request, "%p", &ptr);
+    if (ret != 1 || ptr == NULL)
+        return;
+    strlcpy(marlin_server.gcode_name, ptr, FILE_NAME_MAX_LEN);
+    _send_notify_event(MARLIN_EVT_GFileChange, 0, 0);
+}
+
+// fill pointer with name of the printing gcode (for WUI), dest param have to be at least 97 chars long!
+void marlin_server_get_gcode_name(const char *dest) {
+    if (dest == NULL)
+        return;
+    char *ptr;
+    int ret = sscanf(dest, "%p", &ptr);
+    if (ret != 1 || ptr == NULL)
+        return;
+    strlcpy(ptr, marlin_server.gcode_name, FILE_NAME_MAX_LEN);
+}
+
```

Comment:
functions marlin_server_get/set_gcode_name and gfileevent was removed and replaced with variables MARLIN_VAR_FILENAME and MARLIN_VAR_FILEPATH, why these functions are back? Please use the variables if it possible.

Reply:
My bad, I wasn't aware of the changes. Im on it.

Reply:
commit `d5ad26b`